src/pyodide.js

```diff
@@ -382,6 +382,15 @@ globalThis.languagePluginLoader = new Promise((resolve, reject) => {
     throw e;
   };
 
+  Module.runPythonSimple = function(code) {
+    let code_c_string = Module.stringToNewUTF8(code);
+    Module._PyRun_SimpleString(code_c_string);
+    Module._free(code_c_string);
+    if (Module._PyErr_Occurred()) {
+      pythonexc2js();
```

Comment:
Do you need `Module._` here? In any case, can you add tests for this code path?

Reply:
Yeah I need `Module._PyErr_Occurred` here. In `EM_JS` (or `EM_ASM`) they offer the shortened version e.g., `_PyErrOccurred` rather than `Module._PyErrOccurred`, but outside of `EM_JS` I need the full path.

I can add a test, sure, though because this is a private entrypoint, this branch reflects a coding error which should be fixed. It's wrong now though, it needs to be `Module._pythonexc2js();`, currently any error in the Python code will probably cause a `ReferenceError`.

Reply:
Okay the error branch works fine as-is. So maybe it'd work to say `if(PyErr_Occurred())`? I'm pretty confused about why it works...

Reply:
Definitely does not work with `if(PyErr_Occurred())`...

Reply:
Oh okay `PyRun_SimpleString` has its own error handling, so that branch never happens.